modules/stochastic_tools/include/vectorpostprocessors/StochasticResults.h

```diff
@@ -33,9 +34,9 @@ class StochasticResults : public GeneralVectorPostprocessor, SamplerInterface
   static InputParameters validParams();
 
   StochasticResults(const InputParameters & parameters);
-  void virtual initialize() override;
-  void virtual finalize() override;
-  void virtual execute() override {}
+  virtual void initialize() override;
+  virtual void finalize() override;
+  virtual void execute() override {}
```

Comment:
This is not a necessary change, but you can always just omit `virtual` since `override` implies `virtual`

Reply:
Didn't know that, but I like seeing `virtual` in front